### DISCUSSION TITLE
Need help with using Display to zoom out and crop

### DISCUSSION THREAD

**Discussion by DusanLesan** [ImageMagick/ImageMagick#4569] [Help]
Hi. I use display command for simple image editing but today I had a challenge where it may be a better way.

I have tried to take one huge image in 1:1 ratio and create the background from it.
The approach I used was: 
- Resized the image to 20% and then again to 80% of that to further adjust.
- After that, I started cropping and full width I was left with after resize was 1719.
- I have used some online ratio calculator to figure out appropriate height and since it had decimal point, I kept decreasing desired width until I got to 1712x963
- After that I just resized up to full hd resolution

The behavior I would desire would be:
- Simply zoom out or fit image to screen preserving the resolution
- When I start cropping, I would like to be able to enter desired box size and then lock the ratio and adjust the size to span the wanted area
- Resize from huge size I would be left with after crop (but in aspect ratio I want) to the size I want

Please tell me there is something I could do to get desired behavior.

**Comment by BillyCroan** [comment]
Facing a similar problem here.  I need to be be able to open an image with the 'display' utility, but zoomed out a little, or zoom to fit the whole image on screen.  It would be extra nice if I could cause the window all the way to the left or right side of the screen as well.  I'm trying to automate/speed up some organization of records that requires me to view the file when OCR is done to check that I pulled the correct values out of it for a filename
